editor/script_editor_debugger.cpp

```diff
@@ -1137,8 +1137,9 @@ void ScriptEditorDebugger::_stack_dump_frame_selected() {
 
 	Dictionary d = ti->get_metadata(0);
 
-	Ref<Script> s = ResourceLoader::load(d["file"]);
-	emit_signal("goto_script_line", s, int(d["line"]) - 1);
+	stack_script = ResourceLoader::load(d["file"]);
+	emit_signal("goto_script_line", stack_script, int(d["line"]) - 1);
+	stack_script.unref();
 
```

Comment:
The member variable `stack_script` to mark up the script target for function which is call from the `goto_script_line` signal. In `ScriptEditor::edit` to check is the script edit command is called from debugger.